noteListView.js

```diff
@@ -0,0 +1,16 @@
+(function(exports) {
+
+  function ListView(list) {
+    this.list = list;
+  }
+
+  ListView.prototype.displayNotes = function() {
+    var historyArray = this.list.history
+    var arrayLength = historyArray.length;
+    for (var i = 0; i < arrayLength; i++) {
```

Comment:
arrayLength should be historyArray.length, because it's more readable. historyArray is specific, whereas arrayLength is generic and could imply *any* array.